index.html

```diff
@@ -1,19 +1,98 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
     <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+  <body>
+    <!-- NAVBAR -->
+    <header>
+      <nav>
+        <div class="navbar">
+          <img src="img/karma-logo.svg" alt="Logo" />
+
+          <ul class="main-menu">
+            <li class="nav-title"><a href="">Meet Karma</a></li>
+            <li><a href="">How it Works</a></li>
+            <li><a href="">Store</a></li>
+            <li><a href="">Blog</a></li>
+            <li><a href="">Help</a></li>
+            <li><a href="">Login</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <!--- END OF NAV BAR -->
+    <section class="cover">
+      <div class="container">
+        <h1>Introducing Karma</h1>
+        <h2>Bring WiFi with you, everywhere you go.</h2>
+        <button>Learn More</button>
+      </div>
+    </section>
+    <!-- PRODUCTS -->
+    <section class="products">
+      <div><h2>Everyone needs a little Karma</h2></div>
+
+      <div class="grid">
```

Comment:
Remember we are targeting semantic tags in HTML so better change divs inside the section into articles,

Reply:
Hi Mathias, those divs are just there to serve as a division between the other elements for use with Grid. I already made use of the section tag and other semantic tags where needed. I don't really have any articles on the website that's why I didn't use it, Did look at where I can use it though.